cli/src/main.rs

```diff
@@ -64,7 +64,13 @@ fn main() -> anyhow::Result<()> {
                 None => { requests::info(&config, &name, all); }
                 Some(InfoCommand::Pkgbuild) => { requests::pkgbuild(&config, &name); }
                 Some(InfoCommand::Build { id }) => { requests::build_info(&config, &name, &id); }
-                Some(InfoCommand::Logs { id }) => { requests::build_logs(&config, &name, &id); }
+                Some(InfoCommand::Logs { id, subscribe, attach }) => { 
```

Comment:
I'd change this interface slightly, as I think that the end user will probably want to subscribe by default. So I imagine it like this, if a user uses the `logs` subcommand, do the following:
- If the user supplied an id (yes, I know, IDs are currently broken but anyways), we query the api for that ID and return the logs if they exist or error otherwise. We ignore the flags here.
- If the user did not supply anything special, we first query the db for the latest logs. If they exist, return them, otherwise (when a build has started but not ended, the api will return a 404), do not return anything but subscribe to the live logs. We can honour the `--attach` flag in this case.
- If the user specifically specifies the `--subscribe` flag, subscribe until the *next* build is done, or honour the `--attach` flag if provided. Do not return the last logs if any.

I think this interface is more convenient and also flexible (enough) for some light scripting. However, let me know what you think of this!


This would also mean we don't return the last logs when subscribing in the broadcast, see there.